fe/fe-core/src/main/java/com/starrocks/catalog/OlapTable.java

```diff
@@ -122,8 +123,6 @@ public enum OlapTableState {
 
     @SerializedName(value = "idToPartition")
     protected Map<Long, Partition> idToPartition = new HashMap<>();
-
-    @SerializedName(value = "nameToPartition")
     protected Map<String, Partition> nameToPartition = Maps.newTreeMap(String.CASE_INSENSITIVE_ORDER);
```

Comment:
If you remove the nameToPartition from the gson serializedName, you should make sure that this feature has not been released, or the nameToPartition will be empty after rollback fe.